src/pages/projectSettings/tabs/GithubCommitQueueTab/getFormSchema.tsx

```diff
@@ -127,7 +127,7 @@ export const getFormSchema = (
               type: "null",
               title: "Trigger Versions With Git Tags",
               description:
-                "If an authorized user pushes a tag that matches a regex, then a version can be created from this alias.",
+                "If an authorized user pushes a tag that matches a specific regex, then a version will be created from this alias.",
```

Comment:
Maybe we can add something like "Note that project admins are not authorized by default; they must be listed below or given permission via Mana. "